langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/Next.java

```diff
@@ -50,14 +57,17 @@ public static Object next(Strand strand, ObjectValue m) {
         IteratorValue xmlIterator = (IteratorValue) m.getNativeData("&iterator&");
 
         if (xmlIterator == null) {
-            xmlIterator = ((XMLValue) m.get("m")).getIterator();
+            xmlIterator = ((XMLValue) m.get(new BmpStringValue("m"))).getIterator();
```

Comment:
Should we use https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/StringUtils.java#L112 instead?

Reply:
fixed